src/main/java/teammates/ui/controller/FeedbackSubmissionEditPageData.java

```diff
@@ -47,13 +168,81 @@ public FeedbackSubmissionEditPageData(AccountAttributes account, StudentAttribut
         
         for (Map.Entry<String, String> pair : emailNamePair.entrySet()) {
             result.add(
-                "<option value=\"" + pair.getKey() + "\"" +
-                (StringHelper.recoverFromSanitizedText(pair.getKey()).equals(currentlySelectedOption)  ? " selected=\"selected\"" : "") +
-                ">" + sanitizeForHtml(pair.getValue()) + 
-                "</option>"
+                "<option value=\"" + pair.getKey() + "\"" 
+                + (StringHelper.recoverFromSanitizedText(pair.getKey()).equals(currentlySelectedOption)  
+                                                ? " selected=\"selected\"" : "")
+                + ">" + sanitizeForHtml(pair.getValue()) 
+                + "</option>"
             );
         }
 
         return result;
     }
+    
+    private void createQuestionsWithResponses() {
+        questionsWithResponses = new ArrayList<StudentFeedbackSubmissionEditQuestionsWithResponses>();
+        int qnIndx = 1;
+        
+        for (FeedbackQuestionAttributes questionAttributes : bundle.getSortedQuestions()) {
+            int numOfResponseBoxes = questionAttributes.numberOfEntitiesToGiveFeedbackTo;
+            int maxResponsesPossible = bundle.recipientList.get(questionAttributes.getId()).size();
+            
+            if (numOfResponseBoxes == Const.MAX_POSSIBLE_RECIPIENTS || numOfResponseBoxes > maxResponsesPossible) {
+                numOfResponseBoxes = maxResponsesPossible;
+            }
+            
+            if (numOfResponseBoxes > 0) {
+                FeedbackSubmissionEditQuestion question = createQuestion(questionAttributes, qnIndx);
+                List<FeedbackSubmissionEditResponse> responses = createResponses(questionAttributes, qnIndx, numOfResponseBoxes);
+            
+                questionsWithResponses.add(new StudentFeedbackSubmissionEditQuestionsWithResponses(
+                                                question, responses, numOfResponseBoxes, maxResponsesPossible));
+                qnIndx++;
+            }
+        }
+    }
+
+    private FeedbackSubmissionEditQuestion createQuestion(FeedbackQuestionAttributes questionAttributes, int qnIndx) {
+        boolean isModeratedQuestion = String.valueOf(questionAttributes.questionNumber).equals(getModeratedQuestion());
+        
+        return new FeedbackSubmissionEditQuestion(questionAttributes.courseId, questionAttributes.questionNumber,
+                                                  qnIndx, questionAttributes.getId(), questionAttributes.getQuestionDetails().questionText, 
```

Comment:
Hmm, this wouldn't be necessary, but should we do it (the ternary operator for a `trueQuestionId` thing) anyway for the benefit of a more informative variable name?
By placing the flow control here, we can use names like `shownQnNo` and `realQnNo` rather than `questionNumber` and `qnIndx` which should make it more readable both in the Java and tag files.